backend/plugin/parser/differ/mysql/differ.go

```diff
@@ -936,6 +944,14 @@ func isReferenceDefinitionEqual(old, new *ast.ReferenceDef) bool {
 	return true
 }
 
+// trimParentheses trims outer parentheses.
+func trimParentheses(expr ast.ExprNode) ast.ExprNode {
+	if node, yes := expr.(*ast.ParenthesesExpr); yes {
+		return trimParentheses(node.Expr)
```

Comment:
Tail recursion!

Reply:
Could we avoid recursion? There is stack size limit in general.

Reply:
Nice catch.